docs/content/getting-started/quick-start-with-kubernetes.md

```diff
@@ -0,0 +1,313 @@
+{: .subtitle }
+
+Using Traefik in a Kubernetes environment could be difficult to start with.
+This Getting-Started guide shows how to run an HTTP application in Kubernetes along
+with Traefik as Ingress Controller.
+
+## Permissions and Accesses
+
+Traefik consumes the Kubernetes API to discover the running services and to unlock its power.
+
+Before being able to consume the Kubernetes API, Traefik needs some permissions.
+This permission mechanism is based on roles defines by the cluster administrator.
+The role is then bound on an account used by an application, Traefik is this case.  
+
+The first step is to create the role.
+The [`ClusterRole`](https://kubernetes.io/docs/reference/kubernetes-api/authorization-resources/cluster-role-v1/#ClusterRole) resource enumerates the resources and actions
+available for the role. In a file called `00-role.yml`, put the following `ClusterRole`:
+
+```yaml tab="00-role.yml"
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: traefik-role
+
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - services
+      - endpoints
+      - secrets
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - extensions
+      - networking.k8s.io
+    resources:
+      - ingresses
+      - ingressclasses
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - extensions
+    resources:
+      - ingresses/status
+    verbs:
+      - update
+```
+
+The next step is to create an account for Traefik.
+In a file called `00-account.yml`, put the following [`ClusterAccount`](https://kubernetes.io/docs/reference/kubernetes-api/authentication-resources/service-account-v1/#ServiceAccount) resource:
+
+```yaml tab="00-account.yml"
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: traefik-account
+```
+
+And then, the last operation consists in binding the role on the account to apply the
+permissions and rules on the latter. In a file called `01-role-binding.yml`, put the
+following [`ClusterRoleBinding`](https://kubernetes.io/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1/#ClusterRoleBinding) resource:
+
+```yaml tab="01-role-binding.yml"
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: traefik-role-binding
+
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: traefik-role
+subjects:
+  - kind: ServiceAccount
+    name: traefik-account
+    namespace: default # Using "default" because we didn't specify a namespace when creating the ClusterAccount.
+```
+
+!!! info
+    
+    `roleRef` is the kubernetes reference to the role created in `00-role.yml`.
+    
+    `subjects` is the list of accounts reference. In the guilde, it
+    only contains the account created in `00-account.yml`
+
+## Deployment and Exposition
+
+The ingress controller (reverse proxy) is a software that runs in the same way as any other application on a cluster.
```

Comment:
This doesn't really explain what is an ingress controller.
WYDT of taking inspiration from or even linking directly to our glossary page?
https://traefik.io/glossary/kubernetes-ingress-and-ingress-controller-101/